.travis.yml

```diff
@@ -12,11 +12,9 @@ env:
   - TOXENV='pep8'
   - TOXENV='isort'
   - TOXENV='docs'
-  - DJANGO='django22' CMS='cms37'
-  - DJANGO='django21' CMS='cms37'
-  - DJANGO='django21' CMS='cms36'
-  - DJANGO='django111' CMS='cms37'
-  - DJANGO='django111' CMS='cms36'
+  - DJANGO='django111' CMS='cms40' FE=1
+  - DJANGO='django21' CMS='cms40' FE=1
+  - DJANGO='django22' CMS='cms40' FE=1
```

Comment:
at least django CMS 3.7 compatibility must be kept, while I am ok to drop any django < 2.2

Reply:
@yakky The sheer amount of differences in Django CMS 4 makes backwards compatibility very difficult because the core of the cms is now extremely simplified, there is no concept of draft live, the relationships for Page, PageContent, and Placeholders are different, and many more, you can see that this is the case with the CK Editor too: https://github.com/divio/djangocms-text-ckeditor/tree/support/4.0.x

Reply:
@vipulnarang95 Can you look at at least trying to make the package backwards compatible? It will be ugly but I guess worth a try to keep @yakky happy.

Reply:
@yakky As this is in it's own support/4.x branch and due to the sheer amount of differences for v4 are we able to drop support of 3.7 just for support/4.x the same as the other packages?

You will see that djangocms-versioning and djangocms-moderation are v4 only.

Reply:
Ok, when a more detailed explanantion and migration guide to django CMS 4 we can decide how to proceed

Reply:
@yakky That is at least a few months away. We are currently blocked by this package. You may need to find other resource to fix this in the future when the resources we have currently are no longer allocated. I will assess how we proceed, we may have to create a new package that offers similar functionality.